### ISSUE TITLE
Staging Review finding: Critical action hover state does not have sufficient contrast

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#106796]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** AEDP
**Product:** Design Patterns
**Feature:** Critical Information and Service List Item Components

## Findings details
**VA.gov Experience Standard - issue:** [User can perceive all meaningful elements.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On `hover`, the critical action component has a background color of #FACE00 and a text color of #005EA2. This has a [color contrast ratio of 4.44:1](https://webaim.org/resources/contrastchecker/?fcolor=005EA2&bcolor=FACE00), which does not meet WCAG AA requirements for normal text (less than 14pt for bold text, which is roughly 18.7px).

<details><summary>Link, screenshot or steps to recreate</summary>

![Hover state of the critical action component. Link text is underlined and blue against a yellow background.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/105516-Brian-4.png?raw=true)

</details>

## Recommended action

Choose a `hover` style such that the contrast between the text and the background meets WCAG requirements. Matching the `focus` style would be sufficient.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 1.4.3 Contrast (Minimum) - Level AA](https://www.w3.org/TR/WCAG21/#contrast-minimum)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by bellepx0** [comment]
Fixed, updated the link text color. It is addressed in an open [PR](https://github.com/department-of-veterans-affairs/component-library/pull/1563) in component-library. 

![Image](https://github.com/user-attachments/assets/eb452f0f-b839-4717-b47d-820e8de3cf65)

**Comment by msbtterswrth** [comment]
confirmed in the PRm lgtm.
